test/drb/test_drb.rb

```diff
@@ -333,6 +340,15 @@ def test_05_too_large_ary
     end
     assert_kind_of(StandardError, exception)
   end
+
+  def test_06_array_operations
+    ary = [1,50,3,844,7,45,23]
+    assert_equal(7, @there.size(ary))
+    assert_equal(973, @there.sum(ary))
+    assert_equal(917217000, @there.multiply(ary))
+    assert_equal(139.0, @there.avg(ary))
+    assert_equal(23.0, @there.median(ary))
```

Comment:
It would be good to also test that the median of an array with an even number of elements. for example the median of [1,2] is 1.5